resources/js/src/app/components/pageDesign/MobileNavigation.js

```diff
@@ -57,23 +71,29 @@ Vue.component("mobile-navigation", {
             const breakpoint = QueryHelper.getCurrentBreakpoint();
             const onMobileBreakpoint = () =>
             {
-                if (this.navigationTree.length <= 0)
-                {
-                    this.$store.dispatch("loadNavigationTree")
-                        .then(() =>
-                        {
-                            this.initNavigation();
-                        });
-                }
+                this.loadInitialTree();
```

Comment:
`this.loadInitialTree()` kann auch direkt an `QueryHelper.addFunction` übergeben werden.

```
QueryHelper.addFunction(this.loadInitialTree.bind(this), this.breakpoints)
```